### PR TITLE
Add form new user

### PR DESCRIPTION
**Pull request by alavieille**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/2035
https://github.com/open-orchestra/open-orchestra/pull/1113
https://github.com/open-orchestra/open-orchestra-user-bundle/pull/136

### CODE REVIEW

File: UserAdminBundle/Controller/Api/UserController.php [hunk 1]
```diff
@@ -26,20 +26,22 @@
...
      * @Config\Method({"GET"})
      *
      * @return FacadeInterface
      */
-    public function showAction($userId)
+    public function showAction($email)
     {
-        $user = $this->get('open_orchestra_user.repository.user')->find($userId);
+        $this->denyAccessUnlessGranted(ContributionActionInterface::READ, UserInterface::ENTITY_TYPE);
+        $user = $this->get('open_orchestra_user.repository.user')->findOneByEmail($email);
```
Comment (ngilain) [inline UserAdminBundle/Controller/Api/UserController.php]: @alavieille : can you check there is a user before passing it to the transformer ? As the email comes from the url there's no warranty that the email match a user.

File: UserAdminBundle/Resources/translations/interface.en.yml [hunk 2]
```diff
@@ -14,6 +14,20 @@ open_orchestra_user_admin:
...
+        title_new: New user
         title_edit: Edit the user
         search_engine:
           search: Search
+        new:
+          contact_information: Contact information
+          first_name: First name
+          last_name: Last name
+          email: Email
+          check_correspondence: Check correspondence
```
Comment (ngilain) [inline UserAdminBundle/Resources/translations/interface.en.yml]: check matching ?

File: UserAdminBundle/Resources/translations/interface.en.yml [hunk 3]
```diff
@@ -14,6 +14,20 @@ open_orchestra_user_admin:
...
+          user_found:
+            title: User found
+            message: The email used match with an user
+            associate: Associate this user at my website
```
Comment (ngilain) [inline UserAdminBundle/Resources/translations/interface.en.yml]: with my website

**Comment by alavieille** [comment]
@ngilain @itkg-nanne pr updated

**Comment by ngilain** [comment]
:+1: when the little frenglish translation is fixed

**Pull request merged**